### PR TITLE
Task/rwa 135

### PR DESCRIPTION
**Pull request by iguannago**
https://tools.hmcts.net/jira/browse/RWA-135

### CODE REVIEW

**Review by lewis150193 — APPROVED** [review]

File: charts/wa-proto-frontend/requirements.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+dependencies:
+  - name: nodejs
+    version: ~1.8.0
+    repository: '@hmctspublic'
+  - name: redis
+    version: "10.0.0"
+    repository: "https://charts.bitnami.com/bitnami"
+    tags:
```
Comment (adusumillipraveen) [inline charts/wa-proto-frontend/requirements.yaml:8]: I usually prefer conditions than tags. Example: https://github.com/hmcts/chart-java/blob/master/java/Chart.yaml#L24

Also, u need to add this to Chart.yaml. you are duplicating nodejs chart here. (that's how v2 works)

Comment (iguannago) [reply charts/wa-proto-frontend/requirements.yaml]: done

File: charts/wa-proto-frontend/values.yaml [hunk 2]
```diff
@@ -8,3 +8,13 @@ nodejs:
...
         - AppInsightsInstrumentationKey
+redis:
+    cluster:
+        enabled: false
+        slaveCount: 0
+    usePassword: false
+    master:
+        persistence:
+            enabled: false
+tags:
```
Comment (adusumillipraveen) [inline charts/wa-proto-frontend/values.yaml:19]:
```suggestion
redis:
  enabled: false
```

File: charts/wa-proto-frontend/values.yaml [hunk 3]
```diff
@@ -8,3 +8,13 @@ nodejs:
...
+    pr: false
```
Comment (adusumillipraveen) [inline charts/wa-proto-frontend/values.yaml:20]:
```suggestion
```

**Pull request merged**